ProcessLib/CMakeLists.txt

```diff
@@ -31,8 +31,7 @@ target_link_libraries(ProcessLib
 
 if (OGS_BUILD_PROCESS_ComponentTransport
     OR OGS_BUILD_PROCESS_RichardsComponentTransport
```

Comment:
Why not remove one of the double lines `OR OGS_BUILD_PROCESS_RichardsComponentTransport`?

Reply:
Removed.